.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,48 @@
+name: CIFuzz
+
+on:
+  pull_request:
+    # Fuzzing is currently only targetting the C extension
+    paths:
+      - "**.c"
+      - "**.cc"
+      - "**.cpp"
+      - "**.cxx"
+      - "**.h"
+
+permissions: {}
+
+jobs:
+  fuzz:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - name: Build Fuzzers
+        id: build
+        uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+        with:
+          oss-fuzz-project-name: "cbor2"
+          language: python
+          dry-run: true
```

Comment:
I enabled `dry-run` for now until we confirm this Action is working as expected.